samples/dashboards/edit-mode/change-close-icon/demo.js

```diff
@@ -0,0 +1,45 @@
+Dashboards.board('container', {
+    editMode: {
+        enabled: true,
+        confirationPopup: {
```

Comment:
I believe this is `confirmationPopup` not `confirationPopup` altho it still doesn't work with this.
Could you please report that in Asana if you haven't already done it?

Reply:
Woops. It's already been reported in Asana in the backlog